### PR TITLE
Functional tests - Fix random error on 'BO> Orders > Create order > Choose shipping' test

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix random error on 'BO> Orders > Create order > Choose shipping' test
| Type?             | refacto
| Category?         | TE 
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | https://github.com/nesrineabdmouleh/PrestaShop/actions/runs/4044016795
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   | no

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/09_chooseShipping.ts [hunk 1]
```diff
@@ -226,6 +226,8 @@ describe('BO - Orders - Create order : Choose shipping', async () => {
     it('should re-check summary block', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/09_chooseShipping.ts]:
```suggestion
    it('should set a gift message and re-check summary block', async function () {
```

**Review by Progi1984 — COMMENTED** [review]
A small wording change.

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/09_chooseShipping.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/09_chooseShipping.ts]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**